examples/great_arc_example.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Drawing and using a Great Arc
+=============================
+
+This example shows you how to define and draw a great arc on an image of the
+Sun, and to extract intensity values along that arc from the data.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from sunpy.coordinates.utils import GreatArc
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+m = sunpy.map.Map(AIA_171_IMAGE)
+
+###############################################################################
+# Let's define the start and end co-ordinates of the arc on the Sun.
+start = SkyCoord(735*u.arcsec, -471*u.arcsec, frame=m.coordinate_frame)
+end = SkyCoord(-100*u.arcsec, 800*u.arcsec, frame=m.coordinate_frame)
+
+###############################################################################
+# Create the great arc between the start and end points.
+great_arc = GreatArc(start, end)
+
+###############################################################################
+# Plot the great arc on the Sun
+fig = plt.figure()
+ax = plt.subplot(projection=m)
+m.plot(axes=ax)
+ax.plot_coord(great_arc.coordinates(), color='c')
+plt.show()
+
+###############################################################################
+# Now we can calculate the nearest integer pixels of the data that correspond
+# to the location of arc.
+pixels = np.asarray(np.rint(great_arc.coordinates().to_pixel(m.wcs)), dtype=int)
```

Comment:
well ok! I did not know `SkyCoord` had a `to_pixel` method.

This might be better as `m.world_to_pixel(great_arc.coordinates())`?